src/main/java/web/pages/flipkart/SearchPage.java

```diff
@@ -0,0 +1,13 @@
+package web.pages.flipkart;
+
+import org.openqa.selenium.WebElement;
+import static com.codeborne.selenide.Selenide.$x;
+
+public class SearchPage {
+    public static WebElement secondpage = $x("//a[@class='_2Xp0TH'][contains(text(),'2')]");
+
+    public SearchPage selectSecondPage() {
```

Comment:
Can we generalise this method like: 
SearchPage selectPage(int pageNo){
}

This would help us to navigate to any page